projects/1.16/assets/inventory-essentials/inventoryessentials/lang/zh_cn.json

```
@@ -0,0 +1,11 @@
+{
+    "text.autoconfig.inventoryessentials.title": "Inventory Essentials",
+    "text.autoconfig.inventoryessentials.option.forceClientImplementation": "强制客户端执行（仅限开发人员）",
+    "text.autoconfig.inventoryessentials.option.forceClientImplementation.tooltip": "试验",
+    "text.autoconfig.inventoryessentials.option.enableSingleTransfer": "启用快速移动单个物品",
+    "text.autoconfig.inventoryessentials.option.enableBulkTransfer": "启用批量快速移动物品",
+    "text.autoconfig.inventoryessentials.option.enableBulkTransferAll": "启用快速移动所有物品",
+    "text.autoconfig.inventoryessentials.option.enableBulkDrop": "启用批量丢弃",
+    "text.autoconfig.inventoryessentials.option.allowBulkTransferAllOnEmptySlot": "允许在空槽位中移动所有物品",
+    "text.autoconfig.inventoryessentials.option.enableShiftDrag": "启用Shift滑动鼠标以移动物品"

```

```suggestion
    "text.autoconfig.inventoryessentials.option.enableShiftDrag": "启用Shift拖动鼠标以移动物品"
```
**[计算机术语](https://www.microsoft.com/en-us/language/Search?&searchTerm=drag&langID=124&Source=true&productid=0)**